package.json

```diff
@@ -6,6 +6,7 @@
     "firebase": "^7.7.0",
     "husky": "^3.1.0",
     "lint-staged": "^9.5.0",
+    "materialize-css": "^1.0.0-rc.2",
```

Comment:
I actually don't mind that you got this to work with the raw `link` and `script` tags in `index.html`. Since you did, I would just remove this so it's not confusing in the future if this is necessary or not.